### DISCUSSION TITLE
[Show Off] gemini-pro-vision vs gpt-4-vision-preview

### DISCUSSION THREAD

**Discussion by H0llyW00dzZ** [ChatGPTNextWeb/NextChat#4088] [Show and tell]
- `gemini-pro-vision`

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/c1e31f14-81c6-4cbb-be26-c4d73ab0e16f)

- `gpt-4-vision-preview`

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/ae51e52d-ea28-47ff-af21-f4c4ba261d66)

**Comment by H0llyW00dzZ** [comment]
> [!NOTE]  
> gpt-4-vision-preview it's bad lmao

**Comment by Mishy8er** [comment]
> > [!NOTE]  
> 
> > gpt-4-vision-preview it's bad lmao

Phoook yea

**Comment by VergilWang15** [comment]
I don't think it's fair in this matter, after all, if Gemini (created by Google) can't recognize the mascot of the Go language (also created by Google), it's a bit embarrassing lol

**Reply by H0llyW00dzZ** [reply]
At least, `gemini-pro-vision` can correctly identify that the image is the gopher mascot of the Go programming language, unlike `gpt-4-vision-preview`, which can't provide the correct answer.

**Reply by VergilWang15** [reply]
‌‌‌‌Haha, just kidding. The image model of OpenAI is indeed not ideal, so I don't use it very often either.

**Reply by H0llyW00dzZ** [reply]
claude - https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/discussions/4088#discussioncomment-8750282

**Comment by H0llyW00dzZ** [comment]
- `claude-3-opus-20240229` 

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/9d9a3311-b7a9-4727-862a-a9c67e936030)

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/18c38b5f-0285-4d09-b833-a53e2b5642da)

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/6df17faa-bb5c-498c-87bb-05be20d06e14)


![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/9e539c7a-28d8-4567-9fa4-f83ec19e5785)

**Reply by VergilWang15** [reply]
‌‌‌The pressure has now shifted to OpenAI
